checkpoints/checkpoint1/todaysvision/website/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
```

Comment:
Seems like you have a lot of files in your images directory that are not being used